sass/themes/donate/donate-cr/components/money-buy-block/_money-buy-block.scss

```diff
@@ -0,0 +1,125 @@
+// Money bu block
+.form__row--moneybuy-block {
+  p {
+    @extend p;
+  }
+  p, .number-input-label {
+    color: $colour-dark-purple;
+  }
+  
+  .form__fieldset {
+    @include clearfix;

```

Comment:
is this clearfix actually doing anything (it's the old `overlflow:hidden` clearfix mixin)

Reply:
@Heleen-cr - yes it as I have re-declared as its own mixin in just the donate folder

Reply:
@import "variables/_mixins";